bigquery/stackoverflow.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+
+# Copyright 2015 Google, Inc.
```

Comment:
Update year.

Reply:
Updated to 2017. Also changed to Google LCC per go/releasing/preparing